vllm/spec_decode/ngram_worker.py

```diff
@@ -7,6 +7,7 @@
 from vllm.sequence import ExecuteModelRequest
 from vllm.spec_decode.interfaces import SpeculativeProposals
 from vllm.spec_decode.proposer_worker_base import NonLLMProposerWorkerBase
+from vllm.spec_decode.selector import DEVICE_TYPE
```

Comment:
This would be confusing because the `DEVICE_TYPE` is determined on the fly. Better to make it a function

```python
@cache
def get_device_type():
    ...
    return device_type
```

But I really doubt that vLLM doesn't have an existing utility that gives you the device type so that you need to make this mapping by yourself...

Reply:
DEVICE TYPE is removed, 'scorer_worker.device' to indicate the type instead